### ISSUE TITLE
Support for evil-mode bindings

### ISSUE THREAD

**Issue by A6GibKm** [lordpretzel/mu4e-views#4]
Support bindings for evil-mode. This can be done by sending a PR to https://github.com/emacs-evil/evil-collection. 

Currently the situation for evil-mode is that the bindings for mu4e-views do not work at all.

**Comment by lordpretzel** [comment]
makes sense, I'll look into this once I find some time. Since, mu4e is already supported I can probably steal the keybindings from there which would make this more tractable.

**Comment by lordpretzel** [comment]
Ok, better late then never. I've submitted a pull request for `evil-collection`: [https://github.com/emacs-evil/evil-collection/pull/429](https://github.com/emacs-evil/evil-collection/pull/429) Let's see what they say. I am not using evil-mode personally, but I did some basic testing that the keybindings work.
